docs/docs/contributing/creating-viz-plugins.mdx

```diff
@@ -35,7 +35,7 @@ version of the template that ships with the version of Superset you are using. T
 can be installed by doing the following:
 
 ```bash
-npm i -g yo
+npm i -g yo (If it is not working, use alternate command "npm i -g @superset-ui/generator-superset")
```

Comment:
In the current state, I believe running `npm i -g @superset-ui/generator-superset` should be the default command and line 39 - 41 should be removed, because the code in the superset repo seems to be behind of what is published on NPM.  

From my understanding, the following happens:
```js
cd superset-frontend/packages/generator-superset // local source code of generator-superset that comes with the superset repo
npm i // install the dependencies for the local code
npm link // globally link the local package so that it can be used anywhere
```

This should work without having to install superset globally from npm via `npm i -g @superset-ui/generator-superset`.
The problem seems to be that the local code in the Superset Repo is behind what is published on NPM. 
The repo code contains version 0.18.25 of the generator:
https://github.com/apache/superset/blob/1573c101a71d6ade1f24ac89f99c4c0998b429c7/superset-frontend/packages/generator-superset/package.json#L2-L3

[According to NPM](https://www.npmjs.com/package/@superset-ui/generator-superset?activeTab=versions) this version is 3 years old and this is probably why id doesn't work with new builds of superset anymore. `npm i -g @superset-ui/generator-superset` on the other hand installs version 2.1.0 which then seems to work. 

However, at a quick glance I couldn't find where the current source code of the generator actually is. Can someone help out and tell me which code is published to https://www.npmjs.com/package/@superset-ui/generator-superset? 🙂

Reply:
The source code is actually what's in the Superset repo. The Superset repo is a monorepo, meaning the contents of the `packages` and `plugins` folders _should_ be periodically published to NPM. We aren't doing this as often as we ought to, but there's been little outcry over it, so it doesn't get prioritized.

If what's in the repo isn't working, then we should fix it, and publish a new version of the generator. 

I'm having second thoughts about even HAVING the generator at this point, since it's painful to maintain. A very vanilla example plugin in a folder in the codebase would be a LOT easier to maintain, as a "starter kit"

In the meantime, I approved this PR as it's far more helpful than harmful, but it needs a rebase to be mergeable.

Reply:
Can I create a new PR for this same one? 